docs/api/expect-typeof.md

```diff
@@ -0,0 +1,491 @@
+# expectTypeOf
+
+- **Type:** `<T>(a: unknown) => ExpectTypeOf`
+
+## not
+
+  - **Type:** `ExpectTypeOf`
+
+  You can negate all assertions, using `.not` property.
+
+## toEqualTypeOf
+
+  - **Type:** `<T>(expected: T) => void`
+
+  This matcher will check, if types are fully equal to each other. This matcher will not fail, if two objects have different values, but the same type. It will fail however, if object is missing a property.
```

Comment:
```suggestion
  This matcher will check if the types are fully equal to each other. This matcher will not fail if two objects have different values, but the same type. It will fail however if an object is missing a property.
```